Makefile

```diff
@@ -14,7 +14,10 @@ all: lint test
 
 COV_REPORT := overalls.coverprofile
 
-DOCKER_IMAGE := uber/fx
+DOCKER_MAKE_CMD ?= test
+DOCKER_GO_VERSION ?= 1.8
+DOCKER_IMAGE := uber/fx-$(DOCKER_GO_VERSION)
```

Comment:
Cuz once I get to it, I will add caching http://rundef.com/fast-travis-ci-docker-build